RebeccaJasper/Lab-Project-19G06
# Display random faces for data collection

# Display random faces for data collection 

For the first part of the data collection, respondents need to see a face for a limited period of time before trying to construct their faces.

## Features
- Navigate to the /photo url see one of 3 GAN photos 
- The image will appear on the screen for only 5 seconds (this time can be edited to a more appropriate duration at a later stage)
- After 5 seconds, the user will be directed to the /statements url to fill out a statement on the person that they've just seen

## Manual Testing
- This was primarily a front-end oriented pull request, with a database call via the server. Therefore, no unit testing was done. 
- In order to test, manually check that the previously mentioned features work
